channels/channels.go

```diff
@@ -49,20 +49,23 @@ func Load(path *string) *Channels {
 
 	chnls := Channels{}
 
+	// REST
+	chnls.REST = &rest.Channel{}
+
 	if err := config.ReadInConfig(); err != nil {
 		switch err.(type) {
 		case viper.ConfigFileNotFoundError:
-			log.Warn("File chn.yml not found, skipping channels")
+			log.Warn("File chn.yml not found, using only REST channel")
```

Comment:
Created the REST channel before loading the channels, in order to use it as the default channel (to use it in cli mode for example).